.github/workflows/stale.yml

```diff
@@ -0,0 +1,42 @@
+on:
```

Comment:
This file should probably be removed given the combined dapr-schedule.yml?

Reply:
Sure.